lib/open_api_spex/store_spec.ex

```diff
@@ -0,0 +1,73 @@
+defmodule OpenApiSpex.StoreSpec do
```

Comment:
I think `Export` is more appropriate to `Store`. We're exporting the spec to a file.

Reply:
Fixed